### ISSUE TITLE
SpyderGUI menu list broken view on large screan (External monitor)

### ISSUE THREAD

**Issue by mnislam01** [spyder-ide/spyder#9037]
### What steps will reproduce the problem?

1. Plug an external monitor larger than 22 inch in size

2. Maximise the Spyder GUI

3. Click on file, Edit etc. 

## Versions

* Spyder version: 3.3.2 
* Python version: 3.7.1
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Linux 4.18.0-16-generic

### Dependencies

```
pyflakes >=0.6.0  :  2.0.0 (OK)
pycodestyle >=2.3 :  2.4.0 (OK)
pygments >=2.0    :  2.3.1 (OK)
sphinx >=0.6.6    :  1.8.2 (OK)
rope >=0.9.4      :  0.11.0 (OK)
jedi >=0.9.0      :  0.13.2 (OK)
psutil >=0.3      :  5.4.8 (OK)
nbconvert >=4.0   :  5.4.0 (OK)
pandas >=0.13.1   :  0.23.4 (OK)
numpy >=1.7       :  1.15.4 (OK)
sympy >=0.7.3     :  1.3 (OK)
cython >=0.21     :  0.29.2 (OK)
qtconsole >=4.2.0 :  4.4.3 (OK)
IPython >=4.0     :  7.2.0 (OK)
matplotlib >=2.0.0:  3.0.2 (OK)
pylint >=0.25     :  2.2.2 (OK)
```

**Comment by ccordoba12** [comment]
> Click on file, Edit etc.

What happens when you do this? At least post a screenshot to understand what's happening.

**Comment by mnislam01** [comment]
Here's a screen record: [link](https://drive.google.com/open?id=1LZZ6Np1uIxjkaOOe2G6SdZoLU_fM4ylB)

**Comment by ccordoba12** [comment]
Please make that link public (it's private right now).

**Comment by mnislam01** [comment]
Opps my bad. It's okay now.

**Comment by ccordoba12** [comment]
Ok, thanks. I don't know what's happening in your case, probably something related to Ubuntu menu bar.

We'll take a look at this in a future release.

**Comment by mnislam01** [comment]
Ok. Cool.

**Comment by bcolsen** [comment]
@mnislam01 What does your screen display setting window look like?  

Could you try this with jupyter-qtconsole to see if the menus are in the same position?

**Comment by mnislam01** [comment]
@bcolsen I am running Spyder3 from Anaconda environment. In my Anaconda environment spyder-karnels version 0.3.0 is installed.

**Comment by bcolsen** [comment]
> Is there any chance that it might causing this problem?

That's not likely. This is most likely a problem with the Anaconda Qt packages and your screen/graphics driver config. Testing this with another Qt  app like jupyter-qtconsole would help narrow it down.

**Comment by mnislam01** [comment]
What's the process of running Spyder from jupiter-qtconsole?

**Comment by bcolsen** [comment]
@mnislam01 I don't want you to run Spyder in a jupyter-qtconsole, but see if the menus on the jupyter-qtconsole behave the same as the menus on Spyder.

To run jupyter-qtconsole:
1. open a new terminal
1. Activate anaconda by running: `export PATH="~/anaconda3/bin:$PATH"`
1. Run: `jupyter-qtconsole`

**Comment by mnislam01** [comment]
Oh ok. `jupyter-qtconsole` menu also breaks in the external monitor.

**Comment by bcolsen** [comment]
> Oh ok. jupyter-qtconsole menu also breaks in the external monitor.

This confirms that this isn't a Spyder bug.

It seems to be reported upstream and fixed in qt version 5.10.

https://bugreports.qt.io/browse/QTBUG-60513?page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel&showAll=true
